### ISSUE TITLE
Original appearance of an uploaded non square album art isn't cropped

### ISSUE THREAD

**Issue by ofsound** [sudara/alonetone#861]
<img width="412" alt="Screenshot 2019-10-21 16 05 24" src="https://user-images.githubusercontent.com/407724/67246814-c0a7e100-f41c-11e9-9d09-ae6848182115.png">

**Comment by Manfred** [comment]
@ofsound Is this still an issue?

**Comment by ofsound** [comment]
@Manfred Yes, right after uploading, when the new image first appears on the page, before the playlist is saved, when it's in this state:

<img src="data/jpeg;base64...">

**Comment by Manfred** [comment]
Ah, that image element is created by the JavaScript based on the file loaded from disk. The easiest way to solve this is probably to change the JavaScript to create a style attribute with a background-image data URL and then style it properly.
